src/routes/ContentDetails/ContentDetails.tsx

```diff
@@ -0,0 +1,50 @@
+import { MainLayout } from '@/components';
+import { findContentValueByLabel } from '@/utils/cms';
+import { Tag, tidyClasses as tc } from '@jod/design-system';
+import { useTranslation } from 'react-i18next';
+import { useLoaderData } from 'react-router';
+import { LoaderData } from './loader';
+
+const ContentDetails = () => {
+  const { data } = useLoaderData<LoaderData>();
+  const { i18n } = useTranslation();
+  const date = data.dateCreated
+    ? new Intl.DateTimeFormat([i18n.language], {
+        dateStyle: 'medium',
+      }).format(new Date(data.dateCreated))
+    : '';
+
+  const taxonomies = data.taxonomyCategoryBriefs?.map((taxonomy) => taxonomy.taxonomyCategoryName);
+  const image = findContentValueByLabel(data, 'Kuva')?.image;
+  const description = findContentValueByLabel(data, 'Kuvaus')?.data;
+  const richTextClasses = tc([
+    '[&_p]:my-5',
+    '[&_li]:my-2',
+    '[&_li]:ml-6',
+    '[&_li]:list-item',
+    '[&_ul]:list-disc',
+    '[&_strong]:font-bold',
+  ]);
+
+  return (
+    <MainLayout navChildren={<div className="bg-todo">TODO: Navigation</div>}>
+      <div className="bg-white p-7 col-span-2 flex flex-col gap-7">
+        <h1 className="text-heading-1">{data.title}</h1>
+        <div className="-mt-6">{date}</div>
+        {image && (
+          <div>
+            <img src={image.contentUrl} alt={image.description} />
+          </div>
+        )}
+        <div className="flex gap-y-1">
+          {taxonomies?.map((taxonomy) => (
+            <Tag key={taxonomy} label={taxonomy} variant="presentation" title={taxonomy} />
+          ))}
+        </div>
+        {description && <div className={richTextClasses} dangerouslySetInnerHTML={{ __html: description }} />}
```

Comment:
Can we trust that CMS-generated content is always safe? Maybe we should still use some sanitization library.

Reply:
Added DOMPurify library https://github.com/cure53/DOMPurify